### PR TITLE
chore: unskip Dropzone tests

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-2705]

### Description

CI tests run multiple times without any flakiness.

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2705]: https://toptal-core.atlassian.net/browse/FX-2705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ozgurkececioglu — APPROVED** [review]
Ran in mac and ubuntu. Saw green (except mac for sidebar but it is not related) 👍

**Review by dmaklygin — APPROVED** [review]

File: cypress/component/Dropzone.spec.tsx [hunk 1]
```diff
@@ -24,12 +24,11 @@ describe('Dropzone', () => {
       </TestingPicasso>
     )
     cy.contains('Max file size').should('be.visible')
-    // TODO: https://toptal-core.atlassian.net/browse/FX-2276
-    // cy.get('body').happoScreenshot()
+
+    cy.get('body').happoScreenshot()
```
[the remark this answers is not in the document]
Comment (OndrejTuma) [reply cypress/component/Dropzone.spec.tsx:28]: Oh I see, the second one was in the skipped test 😁

[the remark this answers is not in the document]
Comment (TomasSlama) [reply cypress/component/Dropzone.spec.tsx:28]: here the screenshot was not taken and in the other test it was skipped before

**Review by OndrejTuma — APPROVED** [review]
I also approved happo tests 👍 - all good

**Review by denieler — APPROVED** [review]
who needed to skip those tests is they just worked 🤔 😁

**Comment by TomasSlama** [comment]
> who needed to skip those tests is they just worked 🤔 😁

@denieler I'm not sure, there must have been a reason, but I tried to run it multiple times and didn't see any flakiness. Maybe upgrade of Happo and Cypress to latest helped 🤷

**Review by augustobmoura — APPROVED** [review]

**Pull request merged**